pkg/topologyinfo/cpus/cpus.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ */
+
+package cpus
+
+import (
+	"strconv"
+
+	"sigs.k8s.io/node-feature-discovery/pkg/topologyinfo/sysfs"
+)
+
+// CPUIdList is a list of CPU IDs (integer core identifier)
```

Comment:
This is great, it would be great to have description like this for other data structures in type.go too

Reply:
done!